code/modules/power/singularity/singularity.dm

```diff
@@ -765,6 +765,8 @@ var/list/global_singularity_pool
 			
 			
 /obj/machinery/singularity/deadchat_controlled/proc/begin_democracy_loop()
+	if(democracy_cooldown <= 10)
```

Comment:
make this just < because otherwise it's wasting an operation setting 10 to 10 over and over